src/test/javascript/spec/component/exam/manage/exam-management.component.spec.ts

```diff
@@ -84,4 +85,53 @@ describe('Exam Management Component', () => {
         // THEN
         expect(service.delete).to.have.been.calledOnce;
     });
+
+    it('Should return true for examHasFinished when component has no exam information ', () => {
```

Comment:
The tests are valid but I am questioning their usefulness somehow. This is a very simple method:
```
    examHasFinished(examId: number): boolean {
        if (this.examIdToExamInformation.has(examId)) {
            return this.examIdToExamInformation.get(examId)!.latestIndividualEndDate.isBefore(moment());
        }
        return true;
    }
```
Writing that many test cases for it seems excessive. What do other reviewers say? @krusche

Reply:
Sadly I'm pretty new to testing so I don't know. The goal was to cover all branches in the function.

Reply:
I think the point was that usually a test covering the most common use case would be enough for a simple method. I don't think we should remove the tests now that we have them already.